crates/api_common/src/site.rs

```diff
@@ -149,17 +131,17 @@ pub struct GetModlog {
   #[cfg_attr(feature = "full", ts(optional))]
   pub community_id: Option<CommunityId>,
   #[cfg_attr(feature = "full", ts(optional))]
-  pub page: Option<i64>,
-  #[cfg_attr(feature = "full", ts(optional))]
-  pub limit: Option<i64>,
-  #[cfg_attr(feature = "full", ts(optional))]
   pub type_: Option<ModlogActionType>,
   #[cfg_attr(feature = "full", ts(optional))]
-  pub other_person_id: Option<PersonId>,
+  pub modded_person_id: Option<PersonId>,
```

Comment:
Changed the name of this everywhere I could, to be more consistent.

Reply:
`target_person_id` seems clearer. And there are still a few left:
```
crates/db_schema/src/schema.rs
519:        other_person_id -> Int4,
529:        other_person_id -> Int4,
540:        other_person_id -> Int4,
552:        other_person_id -> Int4,
630:        other_person_id -> Int4,
```

Reply:
That's only in the schema. I'm going to change the `modded_person_id` to `other_person_id` so I don't have to mess with schema changes.

Reply:
We should keep api and db schema consistent, otherwise it can be confusing.